src/golang.org/exercises/jewels-and-stones/jewels-and-stones.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+)
+
+func numJewelsInStones(J string, S string) int {
```

Comment:
When identifiers begin with a capital letter in Go, that typically means they are public and available for use outside the package.  Function parameters are always local and cannot be exported.  Thus I'd suggest making them lower case.

Reply:
Updated to lowercase in commit https://github.com/kcarrel/Golang-Training-Notes/pull/1/commits/0722bd650a57c4f9ee8e1a4801c15decc3142b5f